### PR TITLE
Create paymentVerificationKey on wallet creation (needed for multi-sig)

### PR DESCRIPTION
**Pull request by rcmorano**

### CODE REVIEW

**Comment by nothingalike** [comment]
@rcmorano after our recent update of determining indexes for signing keys, we will need to update this PR

**Comment by rcmorano** [comment]
I think it's ready now :D

File: src/services/wallet.service.js [hunk 1]
```diff
@@ -84,7 +88,24 @@ export async function createWallet(network, name, mnemonic, passphrase) {
...
+    const paymentSKeyPath = path.resolve(walletDir, paymentSigningKeyFile);
+    const paymentSigningKey = getBufferHexFromFile(paymentPrv.stdout).slice(0, 128) + getBufferHexFromFile(paymentPub.stdout);
+    const paymentSKey = `{
+        "type": "PaymentExtendedSigningKeyShelley_ed25519_bip32",
+        "description": "Payment Signing Key",
+        "cborHex": "5880${paymentSigningKey}"
+    }`
+    // public [extended] verification key
+    const extendedVerificationKeyPath = path.resolve(walletDir, extendedVerificationKeyFile);
+    const verificationKeyPath = path.resolve(walletDir, verificationKeyFile);
```
Comment (nothingalike) [inline src/services/wallet.service.js:107]: @rcmorano  are any of these keys sensitive?

Comment (rcmorano) [reply src/services/wallet.service.js:107]: `paymentSkey`, that's why we cleanup [here.](https://github.com/nothingalike/lift-wallet/pull/9/files#diff-76115ce7d41e2ae056afea681e93a327959d67151b960e6fe2e76da1aa7b62f1R154)

**Pull request merged**